### PR TITLE
Update runtime and libraries

### LINKED ISSUE #6: update KDE runtime to 5.15

### PR DESCRIPTION
**Pull request by lionirdeadman**
Fix #6

### CODE REVIEW

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 1]
```diff
@@ -19,11 +21,11 @@ cleanup:
   - /share/aclocal
 modules:
   - name: exiv2
-    buildsystem: cmake
+    buildsystem: cmake-ninja
     sources:
       - type: git
         url: https://github.com/Exiv2/exiv2/
-        commit: e7ffd83af29187190da7f98dcbca8a4d70e19582
+        commit: 15098f4ef50cc721ad0018218acab2ff06e60beb
```
Comment (hfiguiere) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml:28]: it would be nice to add the tag so that it's easy to know which version.

**Comment by lionirdeadman** [comment]
cc @fcomida

**Comment by anxolerd** [comment]
I am currently on the road. Will take a look on Monday when I get home

**Review by anxolerd — APPROVED** [review]
Looks good to me

**Pull request merged**